Sources/Zip/ArchiveFile.swift

```diff
@@ -1,5 +1,5 @@
 import Foundation
```

Comment:
I wonder if we could start using `FoundationEssentials` here

Reply:
I have to check `#if swift(>=6.0)` to `import` it and use the old `Foundation` otherwise, right?

Reply:
I think it should just be available. I mean if you're not using Swift 6 locally yourself and it works it should be fine

Reply:
I've discovered the proper way to include FoundationEssentials and did that in any file where we don't need the whole Foundation